students/rgpag/lesson07/html_render.py

```diff
@@ -0,0 +1,123 @@
+class Element (object):
+    tag_name = ""
+    indent = "    "
+
+    def __init__(self, content=None, **kwargs):
+        self.content = []
+        self.attr_dict = kwargs
+        self.attr_str = ""
+        for k, v in self.attr_dict.items():
+            self.attr_str = self.attr_str+("{}= '{}' ".format(k, v))
```

Comment:
`attr_str` can be local var to `render` method since it is not used in any other methods and specific to render functionality.